### ISSUE TITLE
Bug: Validator does not recognize 'textfield' appearance property

### ISSUE THREAD

**Issue by tmb-github** [w3c/css-validator#309]
See [MDN article on appearance](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance)

```
 /*
It's unaware of the textfield appearance property:
https://developer.mozilla.org/en-US/docs/Web/CSS/appearance
It's also unaware of the -webkit-appearance and -moz-appearance prefixes, which are also documented in the MDN article.
*/
[type=search] {
    -webkit-appearance: textfield;
    -moz-appearance: textfield;
    appearance: textfield;
}
```

**Comment by ylafon** [comment]
See d84e6d267c7ed96c2db23d8b7275cfb307d04694, online as soon as all properties  from the same specification are updated

**Comment by ylafon** [comment]
Fixed by #311
